web/app/models/DrivingSession.js

```diff
@@ -0,0 +1,14 @@
+// grab the mongoose module
+var mongoose = require("mongoose");
+
+// define our driving session model
+// module.exports allows us to pass this to other files when it is called
+var DrivingSessionSchema = new mongoose.Schema({
+    startTime: { type: Number, required: true },
```

Comment:
If mobile is sending over a string representation of time, then the schema can be changed to match that. But if we want a more exact format then we should discuss this more.